### DISCUSSION TITLE
Accessibility Audits with Todd Libby (Lunch & Learn October 15, 2021)

### DISCUSSION THREAD

**Discussion by danieltott** [Virtual-Coffee/virtualcoffee.io#439] [Events]
Accessibility auditing may seem daunting to some or some people may not know where to start. Todd will take you through his auditing process and offer some tips and tricks, workflow ideas, and tools to use so you can do your own accessibility audits.

Thank you to Todd Libby (@colabottles) for this talk! 

The recording of the video will be posted very soon!

#### Todd Week!
- [Todd's VC Podcast Episode](https://virtualcoffee.io/podcast/0402-todd-libby/)
- [Todd on the VC Stream doing an a11y audit for vc.io](https://www.youtube.com/watch?v=8R2Zv8Ic0Cc&t=1790s)

#### Things/People mentioned in the episode:

- [Making A Strong Case for Accessibility](https://www.smashingmagazine.com/2021/07/strong-case-for-accessibility/) by Todd Libby
- [Front Page Express](https://en.wikipedia.org/wiki/Microsoft_FrontPage)
- [Designing with Web Standards](https://en.wikipedia.org/wiki/Designing_with_Web_Standards)
- [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Cass Faris](https://twitter.com/cassandrafaris)
- [Chris DeMars](https://twitter.com/saltnburnem)

#### A11y Learning Resources:

- [Introduction to Web Accessibility](https://www.edx.org/course/web-accessibility-introduction) on edX
- [WebAIM](https://webaim.org/)
- [TPGi](https://www.tpgi.com/)

#### Tools:

- [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org/extension/)
- [axe accessibility toolkit](https://www.deque.com/axe/)
- [pa11y command line](https://pa11y.org/)
- [Bookmarklets by Ian Loyyd and Paul J Adams](https://pauljadam.com/bookmarklets/tables.html)
- [Microsoft a11y insights](https://github.com/microsoft/accessibility-insights-web)
- [Firefox a11y devtools](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector)
- [Lighthouse automated testing](https://developers.google.com/web/tools/lighthouse)
- [Contrast Ratio tool by Lea Verou](https://contrast-ratio.com/)
- [Colour Contrast Analyser](https://www.tpgi.com/color-contrast-checker/)

#### Todd's steps for manual testing:

1. Mouse only
2. Keyboard only
3. Squint test
4. [Screen readers](https://www.afb.org/blindness-and-low-vision/using-technology/assistive-technology-products/screen-readers)
5. [Voice over in Safari](https://webaim.org/articles/voiceover/)
6. [NVDA](https://www.nvaccess.org/) and [JAWS](https://www.freedomscientific.com/products/software/jaws/) on Windows
7. [Orca](https://help.gnome.org/users/orca/stable/introduction.html.en) on Linux

**Comment by bogdaaamn** [comment]
Recording available here: https://www.youtube.com/watch?v=BhUtMZtv-DQ
